.github/workflows/compatability-tests.yaml

```diff
@@ -0,0 +1,300 @@
+name: Compatability Tests
+
+on:
+  pull_request:
```

Comment:
Another option is to trigger this only on push to `master`. The incentive is to save resources, this action alone is probably ~half of an Esti run.

Reply:
I like the option of running only on push to master, I'm thinking about how we can enforce a failure is handled - maybe we can automate a bug creation and assign it to the pr owner? 

Another suggestion is to run it only on pull requests that include changes to hadoopfs and lakeFS API client and on push to master.

Reply:
Yeah, I was also thinking of sending a war-room slack message when failure happens.
Regarding your other suggestion, that won't catch all breaking changes. Consider
```
func (c *Controller) PutObject(...) (...){
    panic("I didn't change the API!!!")
}
```

Reply:
I know that it won't catch all breaking changes but will catch some of them before they are merged

Reply:
But then we're sacrificing coverage. Our coverage is already limited as the Sonnets app is far from covering the entire lakeFS API. I prefer the tests we have for the endpoints is running and catching breaking changes instead of skipping them.